src/components/ble/NimbleController.cpp

```diff
@@ -227,6 +227,9 @@ int NimbleController::OnGAPEvent(ble_gap_event* event) {
         bleController.Disconnect();
         fastAdvCount = 0;
         StartAdvertising();
+
+        systemTask.PushMessage(Pinetime::System::Messages::BleDisconnectAlert);
```

Comment:
I suggest that we call the message just `BleDisconnect` for cleaner semantics. The `alert`-part is not relevant here.